src/Common/Auth/OAuth2KeyConfig.php

```diff
@@ -124,8 +137,14 @@ public function configKeyPairs(): void
                 "encrypt_key" => true,
                 "encrypt_key_cipher" => OPENSSL_CIPHER_AES_256_CBC
             ];
-            $keys = \openssl_pkey_new($keysConfig);
+            $msg = getenv('OPENSSL_CONF');
```

Comment:
@sjpadgett , Just checking on this line above since don't see where this variable is used (except for being written over below).
Also thinking of doing a bit of work on the configKeysPairs function (remove the pieces that create things things are doing this nicely in the new function you created; then can avoid having rather complicated code in 2 different places). Just making sure you are ok with it. Will promise not to break anything, just going to remove unused code :)